butterknife/src/main/java/butterknife/ButterKnife.java

```diff
@@ -187,17 +187,17 @@ public static Unbinder bind(@NonNull Object target, @NonNull View source) {
 
   @Nullable @CheckResult @UiThread
   private static Constructor<? extends Unbinder> findBindingConstructorForClass(Class<?> cls) {
-    Constructor<? extends Unbinder> bindingCtor = BINDINGS.get(cls);
-    if (bindingCtor != null) {
+    if (BINDINGS.containsKey(cls)) {
       if (debug) Log.d(TAG, "HIT: Cached in binding map.");
-      return bindingCtor;
+      return BINDINGS.get(cls);
```

Comment:
I don't like that this requires a double hash lookup in the common case. Can we revert the logic and only check `containsKey` if `get` returns `null`?